recipes/gnucobol/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
```

Comment:
Run this in verbose strict mode please.
```suggestion
set -exuo pipefail
```